packages/jaspr_router/lib/src/misc/extensions.dart

```diff
@@ -36,26 +36,24 @@ extension GoRouterHelper on BuildContext {
   ///
   /// See also:
   /// * [push] which pushes the given location onto the page stack.
-  /// * [pushReplacement] which replaces the top-most page of the page stack but
   ///   always uses a new page key.
   void replace(String location, {Object? extra}) => Router.of(this).replace(location, extra: extra);
 
   /// Replaces the top-most page with the named route and optional parameters,
   /// preserving the page key.
   ///
   /// This will preserve the state and not run any page animation. Optional
-  /// parameters can be providded to the named route, e.g. `name='person',
+  /// parameters can be provided to the named route, e.g. `name='person',
   /// params={'fid': 'f2', 'pid': 'p1'}`.
   ///
   /// See also:
   /// * [pushNamed] which pushes the given location onto the page stack.
-  /// * [pushReplacementNamed] which replaces the top-most page of the page
```

Comment:
@schultek `pushReplacementNamed` does not exist, so I removed these stale docs that reference it.